Sources/PointFree/Database.swift

```diff
@@ -99,7 +123,10 @@ private func upsertUser(withGitHubEnvelope envelope: GitHub.UserEnvelope) -> Eit
       envelope.gitHubUser.name
     ]
     )
-    .flatMap { _ in fetchUser(gitHubAccessToken: envelope.accessToken) }
+    .flatMap { _ in
+      AppEnvironment.current.database
+        .fetchUserByGitHub(envelope.gitHubUser.id)
```

Comment:
oh this was a bit of a gotcha. we gotta make sure to always use the app environment's functions and never directly use the `private` ones in this file.